llpc/tool/llpcCompilationUtils.cpp

```diff
@@ -274,14 +267,18 @@ Expected<std::string> assembleSpirv(const std::string &inFilename) {
   if (binSize < 0)
     return createResultError(Result::ErrorInvalidShader, Twine("Failed to assemble SPIR-V: \n") + log);
 
-  fwrite(spvBin.data(), 1, binSize, outFile);
+  // Caller is responsible for calling delete (note: this will normally happen
+  // as part of cleanupCompileInfo)
```

Comment:
For calling `delete` or `delete[]`? I think it'd be better to be specific, since we ran into issue with new/delete.
Also nit: missing `.` at the end of the sentence.

Reply:
ok - I'll change it.